src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/DeleteAttachmentDtoValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using FluentValidation;
+
+namespace Equinor.ProCoSys.IPO.WebApi.Controllers.Invitation
+{
+    public class DeleteAttachmentDtoValidator : AbstractValidator<DeleteAttachmentDto>
+    {
+        public DeleteAttachmentDtoValidator()
+        {
+            RuleFor(x => x.RowVersion)
```

Comment:
We're no longer doing dto validators I thought? All validation happening in the same spot.